DeltGeometriFelleskomponent.Tests/GeometryEditTest.cs

```diff
@@ -130,4 +195,130 @@ public void InsertsNodeOnLineIndex3()
         Assert.Equal(LineFeature.Geometry.Coordinates[2].X, res.Geometry.Coordinates[2].X);
         Assert.Equal(100, res.Geometry.Coordinates[3].X);
     }
+
+
+
+
+    [Fact]
+    public void DeletesNodeOnLineWithWithNewFeature()
+    {
+        var res = GeometryEdit.EditObject(new EditLineRequest()
+        {
+            AffectedFeatures = new List<NgisFeature>(),
+            Feature = NgisFeatureHelper.Copy(LineFeature),
+            NewFeature = NgisFeatureHelper.Copy(LineFeatureSans1),
+            Edit = new EditLineOperation()
+            {
+                Operation = EditOperation.Delete
```

Comment:
For, man burde vel egentlig ikke trenge å si at dette er en delete? Det burde vi vel klare å analysere oss frem til?

Reply:
Implemented